test/check-application

```diff
@@ -1518,6 +1564,25 @@ class TestFiles(testlib.MachineCase):
         b.key("ArrowLeft")
         b.wait_visible("[data-item='file0'].row-selected")
 
+        # Go up and down in directory hierarchy
+        b.click("[data-item='testdir']")
+        b.key("ArrowDown", modifiers=["Alt"])
+        self.assert_last_breadcrumb("testdir")
+        b.key("ArrowUp", modifiers=["Alt"])
+        b.wait_visible("[data-item='testdir']")
+
+        # Manually edit path
+        b.key("L", modifiers=["Control"])
+        b.input_text("/home/admin/anotherdir")
+        b.key("Enter")
+        self.assert_last_breadcrumb("anotherdir")
+        b.go("/files#/?path=/home/admin")
```

Comment:
Ideally this asserts we loaded the page so the test won't get too flaky.

So I'd recommend:

```
        self.assert_last_breadcrumb("admin")
```